tests/test_transactions_api.py

```diff
@@ -212,6 +212,27 @@ def test_column_formatting(self):
         }))
 
         data = json.loads(response.data)
-        assert response.status_code == 200
+        assert response.status_code == 200, response.data
         assert len(data['data']) == 1
         assert data['data'][0]['event_id'] == first_event_id
+
+    def test_apdex_function(self):
```

Comment:
Could you please also add a quick test in test_util.py that only calls this method, or just as a test case in test_complex_conditions_expr? We have often a hard time troubleshooting this test when it breaks since it does not give a lot of useful signal.